controllers/renderphase_controller.go

```diff
@@ -109,27 +101,7 @@ func (r *HelmChartInstallReconciler) Reconcile(req ctrl.Request) (ctrl.Result, e
 	return ctrl.Result{}, r.render(ctx, i)
 }
 
-func (r *HelmChartInstallReconciler) setup(ctx context.Context, stack *helmv1alpha1.HelmChartInstall) error {
-	return nil
-}
-
-func (r *HelmChartInstallReconciler) render(ctx context.Context, claim *unstructured.Unstructured) error {
-	// Steps to rendering:
-	// - Grab the original claim
-	// - Load the stack's stack.yaml configuration
-	// - From the stack.yaml, load the stack's configuration to be processed
-	// - Process the configuration as specified
-	// - Grab the output / result, and Apply the output or report the failure
-	// - Report the result to the status of the original claim
-	//
-	// Other steps:
-	// - Watch for new types being provided by stacks
-	// - When there is a new type provided, watch for instances being created
-	// - When an instance is created, feed it into the renderer
-	//
-	// TODO This could use a bit of refactoring so it flows more nicely
-
-	// configuration is a typed object
+func (r *RenderPhaseReconciler) render(ctx context.Context, claim *unstructured.Unstructured) error {
```

Comment:
Hmmm yeah. Do you have any ideas for alternatives yet? I have been using `claim` as a stand-in for now because conceptually it is similar to a claim, in the sense that it's created by a user, and then interpreted by a controller which creates some other objects as a result. I've been keeping it in the back of my mind as a potential issue, but I haven't spent a whole lot of time trying to come up with something better yet.